### PR TITLE
feat: add arb1 images for several tokens

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Adding images for all tokens that had it missing from this list https://tokenlists.org/token-list?url=https://raw.githubusercontent.com/cowprotocol/token-lists/113aa571bc7ed208d7ed76e4b7481aeb5b107178/src/public/ArbitrumOneCoingeckoTokensList.json

Some are terrible, and they were the best I could find.
Optimized all of them.

And 2 of them are actually jpgs pretending to be pngs 🤫 

# Testing

Visually check them

Once merged, they can be added to https://github.com/cowprotocol/token-lists/blob/aa4bc7cb7d413bafcd23f25bbfc005e66b3c483a/src/scripts/arbitrum/tokens-with-liquidity-arbitrum.csv

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
I checked the first 8 addresses, they matched, then got lazy and trusted your good skills

This one, is a bit low quality:
`src/public/images/42161/0x431402e8b9de9aa016c743880e04e517074d8cec/logo.png`

**Comment by alfetopito** [comment]
> 0x431402e8b9de9aa016c743880e04e517074d8cec

I got this one from arbiscan https://arbiscan.io/token/0x431402e8b9dE9aa016C743880e04E517074D8cEC.
That's because the one on uniswap https://raw.githubusercontent.com/Uniswap/assets/master/blockchains/ethereum/assets/0x584bC13c7D411c00c01A62e8019472dE68768430/logo.png doesn't match the project's design https://www.hegic.co/
Also could not find anything in their website about a media kit or similar 🤷

**Comment by alfetopito** [comment]
Found a better one that look similar

File: src/public/images/42161/0x88a269df8fe7f53e590c561954c52fccc8ec0cfb/logo.png
Comment (shoom3301) [inline src/public/images/42161/0x88a269df8fe7f53e590c561954c52fccc8ec0cfb/logo.png:1]: Would be great to optimize the images.
https://tinypng.com/

this one can be 50% less

image

**Review by shoom3301 — APPROVED** [review]

File: src/public/images/42161/0x88a269df8fe7f53e590c561954c52fccc8ec0cfb/logo.png
Comment (alfetopito) [reply src/public/images/42161/0x88a269df8fe7f53e590c561954c52fccc8ec0cfb/logo.png:1]: I already did, using the same methods we had before:
1. imageoptim
2. imagemagick: convert <source> -depth 7 -resize 256x -posterize 48 <destination>

**Pull request merged**